pkg/ps/ps.go

```diff
@@ -241,6 +241,11 @@ func ListContainerBatch(rt *libpod.Runtime, ctr *libpod.Container, opts entities
 			UTS:    uts,
 		}
 	}
+
+	if hc, err := ctr.HealthCheckStatus(); err == nil {
+		ps.Status = hc
```

Comment:
We should at least warn on error.

Reply:
Adding warning-log changes "ps" output, which results in failure of other tests.

Reply:
Then at least add it to Debug.

Reply:
Added debug-log.